includes/admin/upgrades/upgrade-functions.php

```diff
@@ -1150,3 +1206,21 @@ function give_v1813_upgrades() {
 	$roles->add_roles();
 	$roles->add_caps();
 }
+
+/**
+ * Correct currency code for "Iranian Currency" in Give setting.
+ * Version 1.8.14 automatic updates
+ *
+ * @since 1.8.14
```

Comment:
@emgk can you update the version throughout to `1.8.17`?

Reply:
Done.